microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -196,6 +196,13 @@ Microsoft Edge lets you download files safely and securely.  To choose where fil
 
 To view the history of your previous downloads, go to `edge://downloads`.  To clear your browsing data and delete your download history, go to `edge://settings/clearBrowserData`.  Deleting your download history from Microsoft Edge does not remove the files from your device.  Deleting downloaded files from your device does not remove the files from your download history.  When you use **InPrivate** browsing or **Guest** mode, the download history from the session is cleared when you close the **InPrivate** or **Guest** windows. The files remain saved on the device.
 
+<!-- ====================================================================== -->
+## Enterprise Security & Compliance
```

Comment:
```suggestion
## Enterprise Security and Compliance

```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

Reply:
Working with the team to understand if this is a brand name. If not, I will make the '&' update and make change Security and Compliance to lowercase